java-lib/src/main/java/com/wavefront/ingester/OpenTSDBDecoder.java

```diff
@@ -14,6 +16,7 @@
  * @author Clement Pang (clement@wavefront.com).
  */
 public class OpenTSDBDecoder implements Decoder {
+  protected static final Logger logger = Logger.getLogger("OpenTSDBDecoder");
```

Comment:
OpenTSDBDecoder.class.getCanonicalName() (this prevent the inevitable renaming of the classes down the line, causing divergence between logs and the class logging it)